src/app/components/applications/SSV/MyAccount/common/ValidatorWhiteHeader/ValidatorWhiteHeader.tsx

```diff
@@ -45,14 +45,14 @@ const ValidatorWhiteHeader = (props: Props) => {
         action: 'click',
         label: 'operator',
       });
-      window.open(`${config.links.EXPLORER_URL}/operators/${props.address}/?version=${config.links.EXPLORER_VERSION}&network=${config.links.EXPLORER_NETWORK}`, '_blank');
+      window.open(`${config.links.EXPLORER_URL}/operators/${props.address}/?version=${config.links.EXPLORER_VERSION}&network=${ENV().NETWORK}`, '_blank');
     } else {
       GoogleTagManager.getInstance().sendEvent({
         category: 'explorer_link',
         action: 'click',
         label: 'validator',
       });
-      window.open(`${config.links.EXPLORER_URL}/validators/${validator.public_key.replace('0x', '')}/?version=${config.links.EXPLORER_VERSION}&network=${config.links.EXPLORER_NETWORK}`, '_blank');
+      window.open(`${config.links.EXPLORER_URL}/validators/${validator.public_key.replace('0x', '')}/?version=${config.links.EXPLORER_VERSION}&network=${ENV().NETWORK}`, '_blank');
```

Comment:
Explorer url already defines which network and version he will work with, he doesn't consider version and network parameters from query string, it can be removed. 
Also, network and version removed from github workflow yaml file, so they will be undefined.

Reply:
done